frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -92,6 +92,36 @@ export const BucketsItem: React.FC<BucketsItemProps> = React.memo(
   ),
 );
 
+export const BucketsTitle: React.FC<BucketsTitleProps> = ({
+  objectsCount,
+  link,
+  error,
+  children,
+}) => {
+  let objectsBody;
+  if (!objectsCount && !error) {
+    objectsBody = <div className="skeleton-text" />;
+  } else {
+    objectsBody = (
+      <div className="co-dashboard-text--small text-secondary">
+        {!error ? formatCount(Number(objectsCount)) : 'Not available'}
+      </div>
+    );
+  }
+  return (
+    <div className="nb-buckets-card__buckets-status-title">
+      {link ? <Link to={link}>{children}</Link> : <div>{children}</div>}
```

Comment:
```suggestion
      {link ? <Link to={link}>{children}</Link> : {children}}
```
Do we need two levels of `<div>`?